commands/utility/wiki.js

```diff
@@ -0,0 +1,45 @@
+const Discord = require('discord.js')
+const got = require('got')
+
+exports.run = async (handler, message, args, pre) => {
+	if (!args.text) {
+		return message.reply('must provide a text argument.')
+	}
+
+	let query = `https://en.wikipedia.org/w/api.php?action=query&prop=pageimages|extracts&titles=${args.text}&exintro=&exsentences=2&redirects=&explaintext=&formatversion=2&piprop=original&format=json`
```

Comment:
`args.text` isn't escaped properly here. So searching for any string that has (for example) a "&" or a "#" in it will break it.